### ISSUE TITLE
Unable to start container in 17.06.0-ce

### ISSUE THREAD

**Issue by RelativeRon** [docker/for-win#869]
### Expected behavior
The container starts
### Actual behavior
COMMAND LINE
PS C:\Users\Ron> docker start vpdev
Error response from daemon: driver failed programming external connectivity on endpoint vpdev (768b83edeb3d3dc770f41dfcb2ac0a459cc450acef29b26a835c1d1cebb218db): Error starting userland proxy: mkdir /port/tcp:0.0.0.0:80:tcp:172.17.0.2:80: input/output error
Error: failed to start containers: vpdev

LOGFILE
[09:48:11.392][NamedPipeClient][Error  ] Unable to send Unmount: The process cannot access the file 'C:\WINDOWS\TEMP\klxxispf.qw1' because it is being used by another process.
[09:48:11.413][ShareHelper    ][Error  ] Failed to unmount C: The process cannot access the file 'C:\WINDOWS\TEMP\klxxispf.qw1' because it is being used by another process.
[09:48:11.414][NamedPipeClient][Info   ] Sending Mount(D, rondell\Ron:**********, Docker.Core.Settings)...
[09:51:12.305][NamedPipeClient][Error  ] Unable to send Mount: The process cannot access the file 'C:\WINDOWS\TEMP\5ncdedz2.um3' because it is being used by another process.
[09:51:12.307][Notifications  ][Error  ] The process cannot access the file 'C:\WINDOWS\TEMP\5ncdedz2.um3' because it is being used by another process.

### Information
  - Diagnostic ID from "Diagnose & Feedback" in the menu.
373FBC67-7F5E-45FD-BB37-ECDF30FFD5C6/2017-06-29_11-11-14
  - host distribution and version (Windows version, build number, etc)
Windows 10 Pro, Docker version 17.06.0-ce, build 02c1d87

### Steps to reproduce the behavior
  1. ... docker start vpdev, but it might be peculiar to my container. Perhaps this happens because I created the container in the previous version of docker, but now in 17.06.0-ce it is failing. I will build the container afresh.... Building it afresh is not working. After entering the docker run command, it just sits there, so far for a very very long time. If I can, I will install the previous version. I need to get this working today.
  2. ... I reset docker to original settings, then installed the image and built the container... successfully. So that was my "fix"

**Comment by jasonbivins** [comment]
Hi @RelativeRon   This was a bug in the way Docker was starting up in, but should be taken care of in the latest Edge build - you might need to use the Reset feature in the GUI for it to take effect though.

**Comment by jasonbivins** [comment]
Hi @RelativeRon 
Are you still seeing this issue in the latest build?

**Comment by RelativeRon** [comment]
Thanks for asking.

I have not used it for the past five days.
Yesterday docker tried to start automatically when I turned on my machine,
and it presented me with an error screen. I did nothing further since I did
not need it yesterday.

Today, docker started automatically just fine. So I started my container --
just fine. Stopped it. Started it again. Stopped it. All is just fine today.
This in spite of the fact that the version and build is the same as when I
had the problem: Windows 10 Pro, Docker version 17.06.0-ce, build 02c1d87

If something else has changed that makes it work better, I'm grateful.

Best,

Ron
